forgerock-openbanking-directory-server/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   mongo:
     container_name: mongo-directory
-    image: mongo:4.2
+    image: mongo
```

Comment:
We want to specify version here surely? Otherwise we don't control when we get a new version and deployments won't be repeatable or predictable??

Reply:
I have delete the version because for some reason that version doesn't work, the docker-compose for that module had the below error:
```bash
:46:20.055 [cluster-ClusterId{value='6033a79b1699b79220ee3419', description='null'}-localhost:27017] INFO  org.mongodb.driver.cluster - Exception in monitor thread while connecting to server localhost:27017
com.mongodb.MongoSocketOpenException: Exception opening socket
```
The module `directory-sample` hadn't the version and worked properly.

Reply:
We can open a Issue to update all mongo version images through all repositories to use the same version.

Reply:
I expect you already had a mongo running. Do a docker ps to see if it was already running.

Reply:
This has always worked, so no reason why it would suddently fail and I don't want to change anything that isn't absolutely necessary or completely understood right now. We need to make things stable and changing things for no good reason isn't making things more stable.

Reply:
THis is only in the directory's docker-compose file, which is not used by any of the integration tests.